### DISCUSSION TITLE
Grid Tracker Connection

### DISCUSSION THREAD

**Discussion by nixfindus** [wavelog/wavelog#360] [Q&A] [answered]
First of all, thanks for the great Job with Wavelog!I use GridTracker in version v1.24.0104 and have set the url https://wavelog.xy.com/, set an API key and the station ID with 1, but I can't get a connection. The Response in GridTracker is "Invalid Response" What is the best way to start the debugging? Thanks!

**Comment by phl0** [comment]
I would say enable development mode and set log threshold to 4. Then have a look at the logfile in application/logs.

**Reply by nixfindus** [reply]
Pleaase a quick hint where to enable der dev mode, thanks

**Reply by phl0** [reply]
Edit index.php or create a file .debug in WL root directory.

**Reply by nixfindus** [reply]
Like this?
![iScreen Shoter - Royal TSX - 240514231350](https://github.com/wavelog/wavelog/assets/6288396/49f9979c-8f57-47ff-bb7f-8e4d3afb7253)

**Comment by phl0** [comment, marked as answer]
I built GridTracker from source to trace down the error. It seems the solution is very simple. Did you configure the complete path tot WL api in the config (including /index.php/api/qso)? I set:

https://URL_TO_WAVELOG/index.php/api/qso

And it seems to work (no QSO logged yet):

![Screenshot from 2024-05-16 10-50-28](https://github.com/wavelog/wavelog/assets/7112907/6bea2d49-2361-4c2b-9607-c37901303ddc)

**Reply by nixfindus** [reply]
Greta thanks! Thats the solution!
